lib/bom.js

```diff
@@ -368,6 +368,11 @@ declare class CloseEvent extends Event {
     wasClean: boolean;
 }
 
+type MessageEventHandler = (event: MessageEvent) => mixed;
+type CloseEventHandler = (event: CloseEvent) => mixed;
+type MessageEventListener = {handleEvent: MessageEventHandler} | MessageEventHandler;
```

Comment:
I never knew about `addEventListener({ handleEvent(e) { ... } });`, we should probably add support for that everywhere.